src/main/java/io/candydoc/ddd/extract_ddd_concepts/DDDConceptFinder.java

```diff
@@ -8,13 +8,17 @@
 import io.candydoc.ddd.model.CanonicalName;
 import io.candydoc.ddd.model.DDDConcept;
 import io.candydoc.ddd.model.Interaction;
+import io.candydoc.ddd.model.PackageName;
+import io.candydoc.ddd.shared_kernel.SharedKernel;
 import io.candydoc.ddd.value_object.ValueObject;
 import java.util.Set;
 
 public interface DDDConceptFinder {
 
   Set<DDDConcept> findDDDConcepts();
 
+  Set<DDDConcept> findDDDConcepts(PackageName packageName);
```

Comment:
PackageName or String ?

I think we should be consistent in the method signatures. Do the other methods should take a PackageName or is this method should take a String ?